### PR TITLE
P2 p nathan read me and additional packs

### PR DESCRIPTION
**Pull request by P2P-Nathan**

### CODE REVIEW

File: NutanixCluster/ReadMe.md
Comment (clarkd) [inline NutanixCluster/ReadMe.md]: Is the bold intentional here and on the above image?

File: QND.Hyperv2012R2/details.json
Comment (clarkd) [inline QND.Hyperv2012R2/details.json]: Could do with reworking.. perhaps "A community-backed Hyper-V 2012 R2 Operations Manager MP"

File: SystemCenterCentral.Utilities.Certificates/details.json
Comment (clarkd) [inline SystemCenterCentral.Utilities.Certificates/details.json]: Odd to see Certificates capitalised and Windows not?

**Review by clarkd — DISMISSED** [review]
A large proportion of the JSON and readme files are oddly showing as a binary content which means they can't be reviewed properly. Could you take a look at this before merging? A few other comments added too.

**Comment by clarkd** [comment]
As an aside, it looks like this PR includes the content of @DavidR91's branch to fix the missing quote issue - there is already a PR for that change (with the required approvals) and so it's odd to see that change here.

File: NutanixCluster/ReadMe.md
Comment (P2P-Nathan) [reply NutanixCluster/ReadMe.md]: Not intentional, per se, the markdown is generated systematically I do not check appearance until it renders in the SCOM console.

**Comment by P2P-Nathan** [comment]
re: PR#3, the code has been merged in now, I pulled it into my local branch as the Community Moderators have only recently been added to GitHub and would not have been available to approve a PR.  The initial plan was to merge that code into my branch and cancel the PR.

re: JSON files showing as binary, not sure on these, all of the file were created and modified via PowerShell, maybe the encoding was set to something odd.

**Side note**
Thank you for the notes and feedback on the PRs.  The Community Moderators group/users will be moving to the repo owner prior to the presentation tomorrow.  With the ownership change the GitHub notifications may change as well.

File: QND.Hyperv2012R2/details.json
Comment (P2P-Nathan) [reply QND.Hyperv2012R2/details.json]: Sounds much better, much of the beta data was part of a copy & paste storm, I'll get this update in place.

**Review by richbenwell — APPROVED** [review]

**Pull request merged**
